inorbit_republisher/inorbit_republisher/republisher.py

```diff
@@ -131,10 +131,15 @@ def callback(msg, repub=repub):
                     pubs[topic].publish("{}={}".format(key, val))
 
         in_topic = repub['topic']
+        # Reads QoS from the topic settings
+        in_qos = repub['qos']
```

Comment:
I suggest that instead of this, which makes it mandatory, let's make it optional and default to 10.
I think you can use a python function to get a value and use a default, I am not sure, but it may be `getattr`, something like `getattr(repub, 'qos', 10)` to say "get the value of qos or use 10 if not specified".

Also, can you add an example qos to the example config?